styles.css

```diff
@@ -0,0 +1,18 @@
+*{
+    padding: 10px;
+    margin: auto;
+}
+
+img{
+    max-width: 100%;
+    max-height: auto;
+    display: block;
+    align-items: center;
+    
+}
```

Comment:
please try as much as possible to remove empty spaces within a class in your code base. it makes your work look messy.